### PR TITLE
Model

### PR DESCRIPTION
**Pull request by shella12**
In this milestone I created a Ruby on Rails application that allows the user to:

- [ ] register and log in, so that the data is private to them.

- [ ] introduce new transactions associated with a category.

- [ ] see the money spent on each category.

- [ ] Created test cases

- [ ] Deployed the app

- [ ] Uploaded [loom video](https://www.loom.com/share/af30a5a49d144b4d82d68d3c84d0e74f) on the features

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,12 @@ The **Budget app** keeps track of your transcations It allows the user to:
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+### Live demo <a name="live-demo"></a>
+
+Click this [live demo](https://budget-app-0yss.onrender.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Whoistolu) [inline README.md]: - [ ] [REQUIRED] It looks like the deployment was not successful. Let's take a look at the issues then kindly redeploy as stated by the project requirement 💪🏽

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @shella12 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._
- [ ] [REQUIRED] Kindly note that it is a requirement to give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

- [ ] [OPTIONAL] You have done a great job attaching the presentation video to the PR description, however, it will be more professional also to add it to a section in the README so the users can easily have access to all the project documentation in just one file.
- [ ] [OPTIONAL] You also did a great job speaking about the part of the project you are most proud of. Next time, you can also go ahead to share the test code during the presentation and also try to keep your camera on during the presentation. All these will help you get used to making professional presentations when you eventually emerge as a developer.
- [ ] [OPTIONAL] It would have also been a great idea to enable `:confirmable` in the user model. The `:confirmable` module in Devise is used to require users to confirm their email address before they can access certain features or functionalities of a web application. Once a user registers for an account, Devise sends a confirmation email to the email address provided by the user. The email contains a unique link that the user must click to confirm their email address. Once the user clicks the link, their account is activated and they can access the features and functionalities of the web application.

https://github.com/shella12/budget-app/blob/b1051206f743dbda1268c86686614e00563881ec/app/models/user.rb#L3

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @shella12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- No Linter error ✔️
- Good PR description 👍🏽
- Good commit message(s) 😉

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shella12** [comment]
Thank you 😄

**Comment by alexander16108** [comment]
@shella12  please finish working on your project before submitting for a review to avoid mix up kk

**Comment by shella12** [comment]
> @Whoistolu  sorry about that, I thought I committed change in the `model branch` but I actually. updated it in. the `dev `branch so had to change otherwise I would require another review

**Pull request merged**